src/config.js

```diff
@@ -1 +1,3 @@
-export default {}
+export default {
+  host: 'https://mig.me'
```

Comment:
If you pass this in in the `init` function it will be much more flexible and can change the location depending on the server you are connecting to. If it's configured like this you are locked to the production server.